app/controllers/keywords_controller.rb

```diff
@@ -1,5 +1,22 @@
 # frozen_string_literal: true
 
+require 'csv'
+
 class KeywordsController < ApplicationController
   def index; end
+
+  def create
+    parse_keywords
+    redirect_to keywords_path
+  end
+
+  private
+
+  def parse_keywords
```

Comment:
This method does the CSV parsing. How about to move it to [service object](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#services), and then call it from the controller? 
like `ServiceClassName.new(csv_file).call`

We usually keep the controller thin/lightweight key moving logic to services/models/helpers etc.
The benefit Service code are
- Code is reusable 
- Independently testable 
- Keeping controller in simple abstraction

Reply:
Good suggestion! Thank you! I will update this